dev-tools/packaging/packages.yml

```diff
@@ -1,9 +1,210 @@
 ---
 
 # This file contains the package specifications for both Community Beats and
-# Official Beats. The shared section contains YAML anchors that are used to
-# define common parts of the package in order to not repeat ourselves.
+# Official Beats.
```

Comment:
Can we just take this opportunity to (finally!) remove Beats references and replace them with Agent? 🙂

Reply:
There is definitely some cleanup to perform here, opened a follow-up issue for that and for removing the unused definitions (we have a lot of leftover Beats specs as well) 
https://github.com/elastic/elastic-agent/issues/7766